### PR TITLE
Use `if __name__ == '__main__':` in server examples docs

### LINKED ISSUE #3775: if __name__ == '__main__' for main examples.
https://github.com/aio-libs/aiohttp-devtools/issues/229

Would be useful if we wrapped `web.run_app(app)` if a main check:

```py
if __name__ == '__main__':
    web.run_app(app)
```

To work with aiohttp-devtools and other such tools.

If you're happy with this I'll submit a PR.

### PR DESCRIPTION
**Pull request by samuelcolvin**
## What do these changes do?

Make it slightly easier to use the default examples with aiohttp-devtools etc.

## Are there changes in behavior for the user?

no

## Related issue number

fix #3775

## Checklist

- [x] Documentation reflects the changes
- [x] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names. 
- [x] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -76,8 +76,9 @@ Client example::
             html = await fetch(session, 'http://python.org')
             print(html)
 
-    loop = asyncio.get_event_loop()
-    loop.run_until_complete(main())
+    if __name__ == '__main__':
+        loop = asyncio.get_event_loop()
+        loop.run_until_complete(main())
```
Comment (webknjaz) [inline docs/index.rst:81]: maybe py3.7-style `asyncio.run()`?

Comment (asvetlov) [reply docs/index.rst:81]: Not yet.
I don't want to reply on multiple nubies complains "why the very first aiohttp example doesn't work on my laptop".

Comment (samuelcolvin) [reply docs/index.rst:81]: agreed.

**Comment by asvetlov** [comment]
Side note:
@webknjaz I constantly see a problem with cherry-picker.
```
andrew•~/projects/aiohttp(master⚡)» cherry_picker 506c244d 3.5                                                             (aiohttp)  [19:34:41]
🐍 🍒 ⛏
Usage: cherry_picker [OPTIONS] [COMMIT_SHA1] [BRANCHES]...
Try "cherry_picker -h" for help.

Error: Run state cherry-picker.state=BACKPORT_COMPLETE in Git config is not known.
Perhaps it has been set by a newer version of cherry-picker. Try upgrading.
Valid states are: BACKPORT_PAUSED, UNSET. If this looks suspicious, raise an issue at https://github.com/python/core-workflow/issues/new.
As the last resort you can reset the runtime state stored in Git config using the following command: `git config --local --remove-section cherry-picker`
```
The version is brand new: 
```
andrew•~/projects/aiohttp(master⚡)» pip show cherry_picker                                                                 (aiohttp)  [19:35:01]
Name: cherry-picker
Version: 1.3.1
Summary: Backport CPython changes from master to maintenance branches.
Home-page: https://github.com/python/core-workflow/tree/master/cherry_picker
Author: Mariatta Wijaya
Author-email: mariatta.wijaya@gmail.com
License: UNKNOWN
Location: /home/andrew/.virtualenvs/aiohttp/lib/python3.7/site-packages
Requires: toml, click, gidgethub, requests
Required-by: 
```

```
andrew•~/projects/aiohttp(master⚡)» cat .git/config                                                                        
...
[cherry-picker]
	state = BACKPORT_COMPLETE
```

**Comment by samuelcolvin** [comment]
what's wrong with squash merge in github?

**Comment by asvetlov** [comment]
I mean backporting to 3.5 branch

**Comment by asvetlov** [comment]
Currently I do backporting by running `cherry_picker <commit_hash> 3.5`.
@Mariatta promised to write github bot for it to backport automatically if the corresponding github label is set. Works like a charm for CPython itself, I love to have such bot for aiohttp

**Comment by Mariatta** [comment]
Sorry about the issue with cherry-picker. It is also a problem in miss-islington, so we reverted back to using earlier cherry-picker 1.2.x.

> @Mariatta promised to write github bot for it to backport automatically if the corresponding github label is set. 

Yes, I will be writing this bot sometime this week or next.

**Comment by Mariatta** [comment]
About cherry-picker issue, I had described it to @webknjaz. He said he will think about it after returning from PyCon 🙂

**Comment by webknjaz** [comment]
@asvetlov, clean the state in Git and use the latest version of cherry-picker, if the one you use creates that state this means that you use an outdated version which has this bug.

**Comment by webknjaz** [comment]
Yes, I'll take a look at addressing this issue.

**Comment by asvetlov** [comment]
I *use* cherry_picker 1.3.1 but it keeps a record in git config after cherry-picking finishing

**Comment by webknjaz** [comment]
@asvetlov did you clean up the local git config _before_ using 1.3.1? Can you confirm that it appears in `.git/config` under the cherry-picker section? We've removed that state identifier from the source code of cherry-picker and I cannot imagine it being recorded into the state storage as it no longer exists in the source code.

**Comment by asvetlov** [comment]
I understand your astonishment but yes.
Now I run `git config --local --remove-section cherry-picker` *before* every `cherry_picker` run.
After the run `state = BACKPORT_COMPLETE` is saved in git config.
The version is 1.3.1

Try yourself

**Comment by webknjaz** [comment]
Wow. I have to check what's _actually_ published on PyPI then.

**Comment by webknjaz** [comment]
Ref: https://github.com/python/core-workflow/issues/320#issuecomment-494790862

**Pull request merged**